metadata.json

```diff
@@ -12,8 +12,8 @@
     {"name":"puppet","version_requirement":">= 3.0.0 < 7.0.0"}
   ],
   "dependencies": [
-    {"name":"puppetlabs/sshkeys_core","version_requirement":">= 1.0.1 <2.0.0"},
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 6.0.0"}
+    {"name":"puppetlabs/sshkeys_core","version_requirement":">= 1.0.1 <2.1.0"},
```

Comment:
These versions don't make sense. The upper bound should be a major release such as `< 5.0.0`.

Are you trying to allow for sshkeys_core to accept v1 and v2 of the module and for stdlib to accept versions 4, 5 and 6?

Reply:
Sorry, I completely missed this response. Will fix it.